.travis.yml

```diff
@@ -26,6 +27,14 @@ jobs:
       services: postgresql
       script: jdk_switcher use oraclejdk8 && ext/travisci/run-core-tests
 
+    # OracleJDK9
+    - stage: ❧ pdb tests
+      env: PDB_TEST=core/oraclejdk9
+      addons:
+        postgresql: *pg-ver
+      services: postgresql
+      script: jdk_switcher use oraclejdk9 && ext/travisci/run-core-tests
+
     # === integration tests
     - stage: ❧ pdb tests
       env: PDB_TEST=int/openjdk8/pup-master/srv-master
```

Comment:
Did you try adding JDK9 integration tests too?  If those also work now, think we'll want them.

Reply:
I'll add them in the next commit